Source/Plugins/Core/com.equella.core/js/tsrc/search/components/SearchResult.tsx

```diff
@@ -141,6 +146,8 @@ export default function SearchResult({
     viewerDetails?: OEQ.MimeType.MimeTypeViewerDetail;
   }
   const renderData = getRenderData();
+  const selectionSessionData = renderData?.selectionSessionInfo;
```

Comment:
Nit: I wonder if doing a change like this would help readability:

```suggestion
  const inSelectionSession: boolean = renderData?.selectionSessionInfo;
```

And then in the one place you need `selectionSessionInfo` as the actual data structure, you simply use `renderData?.selectionSessionInfo`.

Reply:
As per discussions this morning, it'd still be good to address this for future readability/maintainability. @PenghaiZhang is going to see what he can come up with.